man/systemd.exec.xml

```diff
@@ -1869,6 +1869,8 @@ SystemCallErrorNumber=EPERM</programlisting>
         variable definitions. The parser strips leading and trailing whitespace from the values of assignments, unless
         you use double quotes (").</para>
 
+        <para>Since v239 systemd supports newlines in double-quoted values set with <varname>EnvironmentFile=</varname></para>
```

Comment:
We don't mention versions things appeared in man pages. Instead, we expect that users get the right man pages for their version.

Reply:
Yep, please don't specify the version. We have NEWS file for this (and people backport patches quite often, so version numbers are not accurate often anyway).